vite.config.development.ts

```diff
@@ -1,11 +1,36 @@
+import { readFileSync as read } from 'node:fs'
 import { defineConfig, mergeConfig, type UserConfigFn, type UserConfig } from 'vite'
 
 import { config as prodConfig } from './vite.config.production'
+const version = JSON.parse(read('./package.json').toString()).version
 // https://vitejs.dev/config/
 export const config: UserConfigFn = (env) => {
   return mergeConfig(
     prodConfig(env),
-    {} satisfies UserConfig,
+    {
+      plugins: [
+        {
+          // replace the `{{.}}` with dev vars
+          // reproducing what the kuma binary does
+          name: 'kuma-index-html-vars',
+          transformIndexHtml: (template) => {
+            return template
+              .replace('{{.BaseGuiPath}}', '/gui')
+              .replace('{{.}}', JSON.stringify({
+                baseGuiPath: '/gui',
+                apiUrl: 'http://localhost:5681',
+                version,
+                product: 'Kuma',
+                mode: 'global',
+                environment: 'universal',
+                storeType: 'postgres',
+                apiReadOnly: false,
```

Comment:
This seems the same as https://github.com/kumahq/kuma-gui/pull/3049/files#diff-47b7ce02023c49f9552984f4026c55914f09f8d403fd8cc08f875f78cc0d1baeR8-R16 - should we apply DRY on it?

Reply:
🤔 I think I'm gonna move this under and `kuma` property in our `package.json` and source it from there. We need to source package.json anyway for the version so we may as well put this JSON in there also if we want it centralized. Will sort today

Reply:
See https://github.com/kumahq/kuma-gui/pull/3049#issuecomment-2441413261